### ISSUE TITLE
Request for IDEAS/FEEDBACK/FEATURE ENHANCEMENT

### ISSUE THREAD

**Issue by algosig** [AI-COMMUNITY-NEPAL/IdeaBase#2]

**Comment by 50417** [comment]
AI chat bot for nepali language

**Comment by algosig** [comment]
@50417 Nice Idea!
We can explore some existing research works and projects that have been done on Nepali Language.
Then we can plan for a project with further discussion among other interested volunteers

**Comment by shakarbhattarai** [comment]
Would love to be able to use AI for Nepali speech recognition.

**Comment by aadeshnpn** [comment]
Yes Lets collect some nepali novels in electronic format. Then we can train some models for those novel to create simple nepali chat bot.
